src/Engine/Audio/SteamAudio.cpp

```diff
@@ -1,59 +1,87 @@
 #include "SteamAudio.hpp"
+#include <assert.h>
 
 #ifdef USINGMEMTRACK
 #include <MemTrackInclude.hpp>
 #endif
 
+using namespace Audio;
+
 SteamAudio::SteamAudio() {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Member variable 'SteamAudio::context' is not initialized in the constructor. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=cppcheck%3AuninitMemberVar)